src/compas/geometry/_primitives/circle.py

```diff
@@ -121,6 +123,25 @@ def to_data(self):
         """
         return self.data
 
+    @property
+    def center(self):
+        """Point: The center of the circle."""
+        return self.plane.point
+
+    @center.setter
+    def center(self, point):
+        self.plane.point = point
+
+    @property
+    def area(self):
+        """Float: The area of the circle."""
+        return math.pi * (self.radius**2)
```

Comment:
it is a technicality, but i try to avoid unnecessary variable lookups and do `from math import pi` instead

Reply:
done